src/code_80850_len_3060.c

```diff
@@ -469,7 +469,7 @@ void show_coin_counter(void) {
 
     if (uiStatus->unk_6C[0] == 0) {
         set_window_properties(0x14, 0x20, 0xa4, 0x40, 0x14, 0x15, coin_counter_draw_content, 0, -1);
-        set_window_update(0x14, &D_80147474);
+        set_window_update(0x14, basic_window_update);
```

Comment:
Does this emit a warning? I thought `&` was required

Reply:
It's not, and imo we should avoid &ing function pointers